app/services/katello/pulp3/repository.rb

```diff
@@ -386,6 +408,49 @@ def lookup_distributions(args)
         distributions_api.list(args).results
       end
 
+      def distributions_list(args = {})
+        distributions_api.list(args)
+      end
```

Comment:
can we make the naming between distributions_list and list_remotes the same?  it doesn't really matter to me which, but I think we use versions_list as well so maybe  OBJECT_list is a simpler standard to go to.   If its too big of a change we can do it later, but i think it will be small